ui/app/components/namespace-picker.ts

```diff
@@ -14,50 +14,53 @@ import { KEYS } from 'core/utils/keyboard-keys';
  * @description component is used to display a dropdown listing all namespaces that the current user has access to.
  *  The user can select a namespace from the dropdown to navigate directly to that namespace.
  *  The "Manage" button directs the user to the namespace management page.
- *  The "Refresh List" button refrehes the list of namespaces in the dropdown.
+ *  The "Refresh List" button refreshes the list of namespaces in the dropdown.
  *
  * @example
  * <NamespacePicker class="hds-side-nav-hide-when-minimized" />
  */
 
+interface NamespaceOption {
+  id: string;
+  path: string;
+  label: string;
+}
+
 export default class NamespacePicker extends Component {
-  @service auth;
-  @service namespace;
-  @service router;
-  @service store;
+  @service declare auth: any;
+  @service declare namespace: any;
+  @service declare router: any;
```

Comment:
There is also a type for Router we've used elsewhere, `type Router from '@ember/routing/router'`
```suggestion
  @service declare router: Router;
```